Corzbank.Web/ClientApp/src/app/header/authorization/login/login.component.ts

```diff
@@ -39,15 +41,17 @@ export class LoginComponent implements OnInit {
   }
 
   login() {
-    this.authenticationService.deleteUser(Guid.parse('d75b0c7d-7e5d-4698-3b85-08da3745942e')).subscribe(x=>{
-      console.log(x);
-      
-    })
-    this.authenticationService.loginUser(this.loginForm.value).subscribe((data: any) => {
-      if (data != null)
+    this.authenticationService.loginUser(this.loginForm.value).subscribe((response: any) => {
+
+      if (response != null){
         this.notificationService.showSuccessfulNotification("User was successfully loged in", '')
+      }
       else
         this.notificationService.showErrorNotification("Invalid data", '')
+
+        this.storageService.setItem(StorageTypeEnum.localStorage, "accessToken", JSON.stringify(response.accessToken));
```

Comment:
Instead of using static string messages, create a Constant class and put all the messages there.

Reply:
export class Constants {
	static readonly AccessTokenKey = 'accessToken';
}

Usage: Constants.AccessTokenKey

Reply:
resolved